tests/std/tests/VSO_0000000_type_traits/test.cpp

```diff
@@ -629,6 +632,10 @@ void test_function_type() {
     STATIC_ASSERT(!is_bounded_array_v<T>);
     STATIC_ASSERT(!is_unbounded_array_v<T>);
 #endif // _HAS_CXX20
+
+#if _HAS_CXX23
+    STATIC_ASSERT(!is_scoped_enum_v<T>);

```

Comment:
I am missing some positive tests

Reply:
Hence the draft PR and text body. Didn't know where the best place to put them in.